src/run_assistant_audio.cc

```diff
@@ -36,22 +35,24 @@ limitations under the License.
 #include "audio_output_alsa.h"
 #endif
 
-#include "google/assistant/embedded/v1alpha2/embedded_assistant.pb.h"
 #include "google/assistant/embedded/v1alpha2/embedded_assistant.grpc.pb.h"
+#include "google/assistant/embedded/v1alpha2/embedded_assistant.pb.h"
 
 #include "assistant_config.h"
 #include "audio_input.h"
 #include "audio_input_file.h"
 #include "base64_encode.h"
 #include "json_util.h"
 
-using google::assistant::embedded::v1alpha2::EmbeddedAssistant;
-using google::assistant::embedded::v1alpha2::AssistRequest;
-using google::assistant::embedded::v1alpha2::AssistResponse;
-using google::assistant::embedded::v1alpha2::AudioInConfig;
-using google::assistant::embedded::v1alpha2::AudioOutConfig;
-using google::assistant::embedded::v1alpha2::ScreenOutConfig;
-using google::assistant::embedded::v1alpha2::AssistResponse_EventType_END_OF_UTTERANCE;
+namespace gae = google::assistant::embedded::v1alpha2;
```

Comment:
I'd probably switch the namespace to `assistant` to be more clear. What do you think?

Reply:
Or should we even do `use namespace` to remove the prefix?

Reply:
I've switched to `assistant`. It is better. However I wouldn't remove the prefix completely to avoid namespace pollution.